net/tstun/wrap.go

```diff
@@ -71,13 +71,12 @@ type Wrapper struct {
 	// buffer stores the oldest unconsumed packet from tdev.
 	// It is made a static buffer in order to avoid allocations.
 	buffer [maxBufferSize]byte
-	// bufferConsumed synchronizes access to buffer (shared by Read and poll).
-	bufferConsumed chan struct{}
-
-	// closed signals poll (by closing) when the device is closed.
-	closed chan struct{}
-	// errors is the error queue populated by poll.
-	errors chan error
+	// bufferC coordinates re-usable buffers between Read and poll.
```

Comment:
"between" suggests to me that bidirectional communication happens. Where does the buffer go after `poll` fills it? How does it get there?

Perhaps rephrase "between Read and poll" to "from Read to poll". If the return path is also interesting and relevant to how this channel functions, also explain it here.

Reply:
I think I would like the docs to specify that this is a buffered channel, because it being buffered is crucial to the semantics. But I'm unsure of myself, because we don't point it out explicitly anywhere else. Your thoughts welcome.